build.gradle

```
@@ -43,7 +47,7 @@ dependencies {
 
 	modApi("me.shedaniel.cloth:cloth-config-fabric:${project.cloth_version}") { exclude(group: "net.fabricmc.fabric-api") }
 
-	modCompileOnly 'curse.maven:wthit-440979:3401714'
+	modCompileOnly "mcp.mobius.waila:wthit-api:fabric-${wthitVersion}"
```

You need the non-API jar to access the internals
```suggestion
	modCompileOnly "mcp.mobius.waila:wthit:fabric-${wthitVersion}"
```